modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -570,6 +572,59 @@ dfx canister request-status 0x58d08e785445dcab4ff090463b9e8b12565a67bf436251d13e
 
 This command displays an error message if the request identifier is invalid or refused by the canister.
 
+== dfx canister set-controller
+
+Use the `+dfx canister set-controller+` command to specify the identity name or principal to use as the new **controller** for a specified canister on the Internet Computer network.
+A controller identity has special rights to manage the canister it controls.
+For example, only a controlling identity can be used to upgrade or transfer cycles to the canister under its control.
```

Comment:
This is true but you can also specify a `custodian` in the wallet canister so multiple identities can transfer cycles to it. Therefore, I would just remove this and just specify that only the controller may install, reinstall, upgrade etc.